src/ExtensionsSample/Samples/DocumentDBSamples.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+
+namespace ExtensionsSample.Samples
+{
+    // To use the DocumentDB samples:
+    // 1. Create a new DocumentDB account
+    // 2. Add the DocumentDB Endpoint Uri to the 'AzureWebJobsDocumentDBUri' App Setting in app.config
+    // 3. Add the DocumentDB Primary Key to the 'AzureWebJobsDocumentDBKey' App Setting in app.config
+    // 4. Add typeof(DocumentDBSamples) to the SamplesTypeLocator in Program.cs
```

Comment:
Yes, much nicer. Still needs a little work here -- all of the host.Call() calls fail if those types aren't actually loaded in the TypeLocator.

But even with that it's a lot easier to deal with. I figure we can incrementally improve the experience as we go forward.

---
In reply to: [56353574](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/47#discussion_r56353574) [](ancestors = 56353574)